tox.ini

```diff
@@ -5,7 +5,6 @@ envlist =
 [testenv]
 deps =
      .[test]
-     zope.testrunner
```

Comment:
`extras = test` would be better than `deps = .[test]`

Reply:
I knew there was another way to spell that but I always forget what it is.